edgevpn/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM alpine AS build
+# renovate: datasource=github-releases depName=mudler/edgevpn
+ENV VERSION=0.30.2
+ARG TARGETARCH
+
+RUN if [[ "$TARGETARCH" = "amd64" ]]; then \
+      export CHECKSUM=45073d7e0fd0a38bfa7fc634fc7f603e6c636e933f8b8ff25a21fa589abce18a; \
```

Comment:
should this be an ARG too ?

Reply:
I was looking at the argocd bundle and checksum is hardcoded there too. We have a renovate comment for the version but I don't think renovate can bump the checksum so I thought we probably do that manually in the end. I couldn't think of another way it could be happening. Any idea?